java/sage/NetworkEncoderManager.java

```diff
@@ -64,7 +64,7 @@ public void detectCaptureDevices(CaptureDevice[] alreadyDetectedDevices)
       }
     }
 
-    if (Sage.getBoolean("network_encoder_discovery", false))
+    if (Sage.getBoolean("network_encoder_discovery", true))
```

Comment:
Oh wait, nevermind...this is actually a separate commit; it was just grouped into one pull request for me. :)